README.md

```diff
@@ -1,11 +1,15 @@
-# Style Resources for Nuxt 2 - Nobody likes extra @import statements!
+# Style Resources for Nuxt 3 - Nobody likes extra @import statements!
 
 [![npm version][npm-version-src]][npm-version-href]
 [![npm downloads][npm-downloads-src]][npm-downloads-href]
 [![Codecov][codecov-src]][codecov-href]
 [![License][license-src]][license-href]
 
->
+If you would like to use the stable version for Nuxt 2, please see the [`mater` branch](https://github.com/nuxt-community/style-resources-module/tree/master)
```

Comment:
```suggestion
If you would like to use the stable version for Nuxt 2, please see the [`master` branch](https://github.com/nuxt-community/style-resources-module/tree/master)
```

Reply:
Also, we might want to say "use major version XYZ" here? 🤔

Reply:
Added description to use `nuxtjs/style-resources@^1`.
Is it OK to create a `v1` branch?
(If OK, I would like you to make it as I do not have the authorization to do so 🙏 .)

https://github.com/nuxt-community/style-resources-module/pull/214/commits/ae115a1385f83c8489adf1ebb513e20c07b9bc40

Reply:
I created `v1` branch.
https://github.com/nuxt-community/style-resources-module/tree/v1